library/src/main/java/com/google/android/exoplayer/MediaFormat.java

```diff
@@ -214,6 +217,29 @@ public static MediaFormat createId3Format() {
         C.UNKNOWN_TIME_US);
   }
 
+  /* package */ MediaFormat(Parcel in) {
```

Comment:
Would be nice if the ordering here is the same as in the constructor below, rather than being slightly different ;).